vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -0,0 +1,97 @@
+#include <CLI/CLI.hpp>
+
+#include <filesystem>
+#include <iostream>
+
+int main(int argc, char** argv)
+{
+    int l_retValue = 0;
+    try
+    {
+        CLI::App l_app{
+            "VPD Command line tool to dump the inventory, read and "
+            "update the keywords,\nto fix mismatch keyword's and reset keyword's value to its default.\n"};
+
+        std::string l_objectPath{};
+        std::string l_recordName{};
+        std::string l_keyword{};
+        std::string l_filePath{};
+
+        auto l_objectOption = l_app.add_option("--object, -O", l_objectPath,
+                                               "Enter the Object Path");
+        auto l_recordOption = l_app.add_option("--record, -R", l_recordName,
+                                               "Enter the Record Name");
+        auto l_keywordOption = l_app.add_option("--keyword, -K", l_keyword,
+                                                "Enter the Keyword");
+
+        auto l_fileOption = l_app.add_option(
+            "--file", l_filePath,
+            "Enter the file name with its absolute path. This option can be used "
+            "in read and write operations. When used in read, the read value will "
+            "be saved to this file and when used in write, the value to be written "
+            "will be taken from this file.");
+
+        auto l_readOption =
+            l_app
+                .add_flag(
+                    "--readKeyword, -r",
+                    "Read the data of the given keyword. { "
+                    "vpd-tool-exe --readKeyword/-r --object/-O "
+                    "\"object-name\" --record/-R \"record-name\" --keyword/-K "
```

Comment:
can we have something like -

vpd-tool --readKeyword --object <path to vpd> --record <record name> --keyword <keyword name> (or)
vpd-tool -r -O <path to vpd> -R <record> -K <keyword>